code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -420,6 +420,12 @@
 	storage_slots = null
 	max_storage_space = 8
 
+/obj/item/storage/belt/fannypack/component
+	name = "component pouch"
+	desc = "A dorky fannypack for keeping small items in. Also stores magickal components!"
```

Comment:
```suggestion
	desc = "A dorky fannypack for keeping small items in. Also stores magical components!"
```

Reply:
I don't think spelling magic the proper way really matters, especially when the application of magic here is less serious in tone.